tests/semantics/test_semantics_models.py

```diff
@@ -1,1013 +1,1013 @@
-                         [semantic_manager.get_datatype("string")])
-
-        # possible classes in relations
-        self.assertEqual(
-            set(obj.objProp3.get_all_possible_classes(include_subclasses=True)),
-            {Class3, Class123, Class13, Class3a})
-        self.assertEqual(
-            set(obj.objProp3.get_all_possible_classes()),
-            {Class3})
-
-        # possible Individuals in relations
-        self.assertEqual(obj.objProp5.get_all_possible_individuals(),
-                         [Individual1()])
-
-    def test__22_get_instances(self):
-        from tests.semantics.models import Class1, Class123, \
-            Gertrude, semantic_manager
-
-        obj = Gertrude()
-        class1 = Class1()
-        class123 = Class123()
-
-        self.assertEqual(
-            set(semantic_manager.get_all_local_instances_of_class(
-                class_name="Class1")),
-            {class1, class123, obj})
-
-        self.assertEqual(
-            set(semantic_manager.get_all_local_instances_of_class(
-                class_name="Class1", get_subclasses=False)),
-            {class1})
-
-        self.assertEqual(
-            set(semantic_manager.get_all_local_instances_of_class(
-                class_=Gertrude)),
-            {obj})
-
-        self.assertEqual(
-            set(semantic_manager.get_all_local_instances_of_class(
-                class_=Class123, get_subclasses=False)),
-            {class123})
-
-    def test__23_valid_instance_ids(self):
-        """Test if inavlid instance ids are prevented and valid ids are accepted"""
-        from tests.semantics.models import Class1
-
-        Class1(id="test")
-        Class1(id="!-+~")
-
-        with self.assertRaises(AssertionError) as context:
-            Class1(id="#'")
-        with self.assertRaises(AssertionError) as context:
-            Class1(id="Kühler")
-
-    def tearDown(self) -> None:
-        """
-        Cleanup test server
-        """
-        try:
-            from tests.semantics.models import semantic_manager
-            self.clear_registry(semantic_manager)
-        except ModuleNotFoundError:
-            pass
-        try:
-            from tests.semantics.models2 import semantic_manager
-            self.clear_registry(semantic_manager)
-        except ModuleNotFoundError:
-            pass
-
-        clear_all(fiware_header=FiwareHeader(
-            service=settings.FIWARE_SERVICE,
-            service_path=settings.FIWARE_SERVICEPATH),
-            cb_url=settings.CB_URL,
-            iota_url=settings.IOTA_JSON_URL)
-
-    @staticmethod
-    def get_file_path(path_end: str) -> str:
-        """
-        Get the correct path to the file needed for this test
-        """
-
-        # Test if the testcase was run directly or over in a global test-run.
-        # Match the needed path to the config file in both cases
-
-        path = Path(__file__).parent.resolve()
-        return str(path.joinpath(path_end))
+# import time
```

Comment:
Yes and no 😄 After the migration, there are some very strange errors, e.g. `'DevicePropertyInstanceLink' is not fully defined`, etc. Since the code is not obvious to me (especially the structure of the modules), I can not fix them in the short term. I think the semantic parts are not the core functionality of filip. Therefore, for the current migration to pydantic V2 I would first leave it behind. After that, we can open another issue, rework `semantics modules`.